src/Compilers/CSharp/Portable/Parser/Blender.Cursor.cs

```diff
@@ -55,7 +55,11 @@ private static bool IsNonZeroWidthOrIsEndOfFile(SyntaxNodeOrToken token)
                 return token.Kind() == SyntaxKind.EndOfFileToken || token.FullWidth != 0;
             }
 
-            public Cursor MoveToNextSibling()
+            /// <summary>
+            /// Returns the cursor of our next non-empty (or EOF) sibling in our parent if one exists, or `default` if
+            /// if doesn't.
+            /// </summary>
+            private Cursor TryFindNextNonZeroWidthOrIsEndOfFileSibling()
```

Comment:
I will approve this PR if you can say this three times fast.

Reply:
This three times fast

Reply:
Damn ... I just got defeated at my own game.